$NetBSD$

#
# allow the spectrum user to be substituted in.
#
# -s to su(1) isn't portable, remove it.
#

--- spectrumctl/spectrum/env.py.orig	2010-07-13 17:38:39.000000000 +0000
+++ spectrumctl/spectrum/env.py
@@ -18,7 +18,7 @@ def get_uid():
 		username = os.environ['SPECTRUM_USER']
 	except KeyError:
 		# otherwise we default to spectrum:
-		username = 'spectrum'
+		username = '@SPECTRUM_USER@'
 	try:
 		return pwd.getpwnam( username ).pw_uid
 	except KeyError:
@@ -32,7 +32,7 @@ def su_cmd( cmd ):
 	TODO: This should be replaced with "drop_privs" or so
 	"""
 	user = pwd.getpwuid( get_uid() ).pw_name
-	return [ 'su', user, '-s', '/bin/sh', '-c', ' '.join( cmd ) ]
+	return [ 'su', user, '/bin/sh', '-c', ' '.join( cmd ) ]
 
 def perm_translate( perm ):
 	"""
